jooq/src/main/java/io/micronaut/configuration/jooq/JooqConfigurationProperties.java

```diff
@@ -0,0 +1,38 @@
+package io.micronaut.configuration.jooq;
+
+import io.micronaut.context.annotation.EachProperty;
+import org.jooq.SQLDialect;
+import org.jooq.tools.jdbc.JDBCUtils;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+@EachProperty(value = "jooq.datasources")
```

Comment:
What is the value of the `@EachProperty` here since it seems there is no settable properties?

Reply:
`sql-dialect` is settable